examples/sign_verify/sign-verify-ecdsa.js

```diff
@@ -0,0 +1,49 @@
+import { crypto } from "k6/x/webcrypto";
+
+export default async function () {
+  try {
+    const keyPair = await crypto.subtle.generateKey(
+      {
+        name: "ECDSA",
+        namedCurve: "P-256",
+      },
+      true,
+      ["sign", "verify"]
+    );
+
+    const data = string2ArrayBuffer("Hello World");
+
+    const alg = { name: "ECDSA", hash: {name: "SHA-256" } };
+
+    // makes a signature of the encoded data with the provided key
+    const signature = await crypto.subtle.sign(alg, keyPair.privateKey, data);
+
+    console.log("signature: ", printArrayBuffer(signature));
+
+    //Verifies the signature of the encoded data with the provided key
+    const verified = await crypto.subtle.verify(
+      alg,
+      keyPair.publicKey,
+      signature,
+      data
+    );
+
+    console.log("verified: ", verified);
+  } catch (err) {
+    console.log("err: " + JSON.stringify(err));
+  }
+}
+
+const string2ArrayBuffer = (str) => {
+  let buf = new ArrayBuffer(str.length * 2); // 2 bytes for each char
+  let bufView = new Uint16Array(buf);
+  for (let i = 0, strLen = str.length; i < strLen; i++) {
+    bufView[i] = str.charCodeAt(i);
+  }
+  return buf;
+};
+
+const printArrayBuffer = (buffer) => {
+  let view = new Uint8Array(buffer);
+  return Array.from(view);
+};
```

Comment:
Having seen this in a bunch of places at this point - maybe move them to a helper functio( in another PR)?

Reply:
I was asking myself the same question but ended up this way since I wanted that example to be "completed" (without the need to have multiple files), just copy/run. Do you think it makes sense?